### PR TITLE
Update docker setup

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**
- There was a bug in the setup
- multiple container down ups runs entrypoint again and again
- Entrypoint had a cd  command

### CODE REVIEW

**Review by MaximilianWagner — APPROVED** [review]
Looks good.

File: docker-compose.yml [hunk 1]
```diff
@@ -25,8 +25,9 @@ services:
     network_mode: "host"
     ports:
       - '8000:8000'
+      - '8001:8001'
```
Comment (diegiesskanne) [inline docker-compose.yml:28]: maybe explain what the ports are for

File: Dockerfile [hunk 2]
```diff
@@ -28,5 +30,6 @@ RUN sudo apt-get update && \
     sudo apt-get install -y swig
 
 EXPOSE 8000
+EXPOSE 8001
```
Comment (diegiesskanne) [inline Dockerfile:33]: same as in docker-compose

**Review by diegiesskanne — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (ashutoshsingh0223) [reply docker-compose.yml:28]: Done

File: Dockerfile (hunk 2 above)
Comment (ashutoshsingh0223) [reply Dockerfile:34]: Done

**Pull request merged**